main_test.go

```diff
@@ -36,10 +36,10 @@ func init() {
 
 	flag.BoolVar(&runAcceptanceTests, "acceptance-tests", false, "set flag when running acceptance tests")
 	flag.StringVar(&cliArgsRaw, "cli-args", "", "for passing urfave/cli args (single string) when golang flags are specified (avoids conflict)")
-	flag.Parse()
 }
 
 func TestRunMain(t *testing.T) {
+	flag.Parse()
```

Comment:
Why?

Reply:
part of the migration to go v.1.13, required.
e.g.: https://golang.org/src/flag/example_test.go
_"// execute, typically at the start of main (not init!):"_